scripts/execute-test

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+TEST=$1
+
+if [ "" == "${TEST}" ]; then
+	echo "You need to specify at least one test"
+	exit 1
+fi
```

Comment:
This runs only the first test. Correct idiom would be

```suggestion
if [ $# -eq 0 ]; then
	echo "You need to specify at least one test"
	exit 1
fi
```

...and see the `npx` block below for how to complete it.